api/provider/task.go

```diff
@@ -3,8 +3,9 @@ package provider
 import "github.com/quintilesims/layer0/common/models"
 
 type TaskProvider interface {
-	Create(req models.CreateTaskRequest) (*models.Task, error)
-	Delete(taskID string) error
-	List() ([]models.TaskSummary, error)
+	// todo: all creates should return a string
+	Create(req models.CreateTaskRequest) (string, error)
 	Read(taskID string) (*models.Task, error)
+	List() ([]models.TaskSummary, error)
+	Delete(taskID string) error
```

Comment:
[super nit] I think last week or so we had decided in favor of alphabetized ordering over CRUD ordering

Reply:
Yes, we did. I'll make sure it's alphabetical here.